### PR TITLE
Stop hardcoding body field type

### LINKED ISSUE #1847: Mark down editor does not work on the edit form
- Laravel Version: 5.5
- Voyager Version: 1.0.0
- PHP Version:7.1
- Database Driver & Version: 5.1

### Description:
The markdown editor is not working on the edit form, it shows only a Textarea field.

### Steps To Reproduce:
1. Create a bread.
2. Set a field as Markdown.
3. Browse the bread and create a new registry, see that the markdown editor is shown.
4. Then, edit the registry and see that the markdown editor is not shown.

### PR DESCRIPTION
**Pull request by marktopper**
Fix #1847

### CODE REVIEW

File: resources/views/posts/edit-add.blade.php [hunk 1]
```diff
@@ -102,7 +102,7 @@
                     <!-- ### CONTENT ### -->
                     <div class="panel">
                         <div class="panel-heading">
-                            <h3 class="panel-title"><i class="icon wb-book"></i> {{ __('voyager::voyager.post.content') }}</h3>
```
Comment (marktopper) [inline resources/views/posts/edit-add.blade.php:105]: The icon `wb-hook` is from way before the `v1.0` release and ends up with given extra un-needed space.

File: resources/views/posts/edit-add.blade.php [hunk 2]
```diff
@@ -111,7 +111,14 @@
                             '_field_name'  => 'body',
                             '_field_trans' => get_field_translations($dataTypeContent, 'body')
                         ])
-                        <textarea class="form-control richTextBox" id="richtextbody" name="body" style="border:0px;">@if(isset($dataTypeContent->body)){{ $dataTypeContent->body }}@endif</textarea>
+                        @php
+                            $dataTypeRows = $dataType->{(isset($dataTypeContent->id) ? 'editRows' : 'addRows' )};
+                            $row = $dataTypeRows->where('field', 'body')->first();
```
Comment (marktopper) [inline resources/views/posts/edit-add.blade.php:116]: NOTE: This is *not* a DB call, it is just filtering the `Collection` to get the correct type.

File: resources/views/posts/edit-add.blade.php [hunk 3]
```diff
@@ -111,7 +111,14 @@
...
+                        @endphp
+                        
+                        <div class="panel-body">
+                            {!! app('voyager')->formField($row, $dataType, $dataTypeContent) !!}
```
Comment (marktopper) [inline resources/views/posts/edit-add.blade.php:120]: Here we allow the `body` field to be any type at all. Not limited to be the `Rich Text Editor`.

**Review by fletch3555 — APPROVED** [review]

**Comment by fletch3555** [comment]
Thanks for the thorough explanation of your changes! Makes sense to me

**Pull request merged**